e2e/cypress/support/api/on_prem_default_config.json

```diff
@@ -231,9 +231,9 @@
     "SupportSettings": {
         "TermsOfServiceLink": "https://mattermost.com/terms-of-use/",
         "PrivacyPolicyLink": "https://mattermost.com/privacy-policy/",
-        "AboutLink": "https://about.mattermost.com/default-about/",
-        "HelpLink": "https://about.mattermost.com/default-help/",
-        "ReportAProblemLink": "https://about.mattermost.com/default-report-a-problem/",
+        "AboutLink": "https://docs.mattermost.com/about/product.html",
```

Comment:
This'll be something we'd also need to change on the server, but changing these will actually break those links for all existing Mattermost instances that haven't customized them. The old about.mattermost.com URLs are saved in those servers' config files, so we need those redirects to still exist.

Even if we could change them for existing installs, I think it would be best to leave them as redirects anyway since we could end up changing the URLs for some of them. The download links for the mobile apps, in particular, will likely change when Mobile v2 comes out

Reply:
@emdecr - Another good point to take into consideration.

Reply:
Similar to my [previous comment](https://github.com/mattermost/mattermost-webapp/pull/9785#discussion_r814310975). Let's revert back to the old links.